ocfweb/docs/docs/services/web/flask.md

```diff
@@ -23,13 +23,21 @@ OCF servers requires only just a little extra configuration.
 
 * run.fcgi:
 
-      #!/usr/bin/env python
+      #!venv/bin/python
```

Comment:
this isn't a pattern i see very often, but it's fine if it works. did you confirm it does with fastcgi?